src/main/java/smartthings/konsumer/KafkaListener.java

```diff
@@ -34,6 +40,7 @@ public KafkaListener(ListenerConfig config) {
 		consumer = Consumer.createJavaConsumerConnector(config.getConsumerConfig());
 		topic = config.getTopic();
 		streamProcessor = new StreamProcessorFactory(config).getProcessor();
+		circuitBreaker = new SimpleCircuitBreaker();
```

Comment:
Can you think of any reason we'd want to customize the circuit breaker? I've thought of turning this into a builder instead of a config object. Feels like we are starting to get close to a place where that makes more sense.  Something we could switch later if we decided it is needed.